workflows/edit_eml/edit_attributelists.Rmd

```diff
@@ -105,13 +113,22 @@ If you have units that are not in the standard EML unit list, you will need to b
 * **name**: Unit abbreviation (e.g. for kilometer `name` = "km")
 * **description**: Text defining the unit (e.g. for kilometer `description` = "1000 meters")
 
-Additionally, `datamgmt::create_attributes_table()` will tell you if each of your units are standard or not. If your unit is not standard, you should use the following code to help auto-generate a custom unit:
+To manually generate the custom units list, create a dataframe with the fields mentioned above. An example is provided below that can be used as a template:
 
-``` {r, eval = FALSE}
-datamgmt::return_eml_units("your_unit")
+```{r, eval = FALSE}
+custom_units <- data.frame(
+    
+  id = c('siemensPerMeter', 'decibar'),
+  unitType = c('resistivity', 'pressure'),
+  parentSI = c('ohmMeter', 'pascal'),
+  multiplierToSI = c('1','10000'),
+  abbreviation = c('S/m','decibar'),
+  description = c('siemens per meter', 'decibar'),
+  
+  stringsAsFactors = FALSE)
 ```
```

Comment:
This is looking really good.  I think you could add a line where you insert this into the EML doc.

Reply:
I actually see that in section 4.3.4 but I think it makes sense to just move it up to the bottom of this section so it's more self-contained